labelbox/client.py

```diff
@@ -1574,6 +1574,30 @@ def get_catalog_slice(self, slice_id) -> CatalogSlice:
         res = self.execute(query_str, {'id': slice_id})
         return Entity.CatalogSlice(self, res['getSavedQuery'])
 
+    def unarchive_feature_schema_node(self, ontology_id: str,
+                                      root_feature_schema_id: str) -> bool:
+        """
+        Returns true if the root feature schema node was successfully unarchived, false otherwise
```

Comment:
Can we specify in the docstring here that only root level feature schemas are unarchivable?

Reply:
[Updated](https://github.com/Labelbox/labelbox-python/pull/939/commits/85e77f6ca6072d90e71ea6cab45aa998cbb6775f)